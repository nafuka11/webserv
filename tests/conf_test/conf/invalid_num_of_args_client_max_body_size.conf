server {
    client_max_body_size 100 200;
}