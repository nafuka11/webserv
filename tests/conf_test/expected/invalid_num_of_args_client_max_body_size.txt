webserv: invalid number of arguments in "client_max_body_size" directive in ./conf/invalid_num_of_args_client_max_body_size.conf:2
